app/index.html

```diff
@@ -4,10 +4,32 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/reset.css">
     <link rel="stylesheet" href="styles/main.css">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <header>
+        <div class="center align-left" id="logo">
+            <span id="logo-text" class="primary-text-color">Welcome</span>
+        </div>
+        <nav class="center align-right">
+            <img id="menu" src="assets/menu_button.png" alt="">
+        </nav>
+    </header>
+    <div class="content">
+        <h2 class="center-text primary-text-color main-heading">Hey there. I&apos;m Mariane <br><span class="secondary-text-color">I&apos;m a software developer</span> </h2>
+
+        <p class="paragraph primary-text-color"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don&apos;t hestiate to contact me.</p>
+
+        <ul class="social-icons">
+            <li><img src="./assets/social-icons/twitter.png" alt=""></li>
+            <li><img src="./assets/social-icons/Linkedin_icon.png" alt=""></li>
+            <li><img src="./assets/social-icons/medium.png" alt=""></li>
+            <li><img src="./assets/social-icons/github.png" alt=""></li>
+            <li><img src="./assets/social-icons/angellist.png" alt=""></li>
```

Comment:
- [ ]  Great job using `ul` in this part, but In the requirements, I believe that you are asked to use the `<a>` tag for social icons, and follow the button styling: `Active`, `Hover`, and `Pressed` (see [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)). If you notice most websites have social icons as buttons and redirect to other links, that kind of UX convention make the navigation in the website way more pleasant and intuitive 😄.

Reply:
Included social-icons in `<a>` tag and are now able to redirect to the links. As far as button styling is concerned if you look at the theme I believe these are not meant to change their formatting or color on hover.